src/openforms/submissions/tests/test_submission_suspension.py

```diff
@@ -141,6 +142,29 @@ def test_email_sent(self):
 
         self.assertIn(defaulttags.date(datetime_removed), email.body)
 
+    @freeze_time("2021-11-15")
+    def test_email_sent_with_custom_configuration(self):
+        config = GlobalConfiguration.get_solo()
+        config.save_form_email_subject = "The Subject"
+        config.save_form_email_content = "The Content"
```

Comment:
I'd like to see some actual custom templates with variables instead of just static content.